shared/build.gradle.kts

```diff
@@ -124,20 +125,25 @@ spotless {
     kotlin {
         custom(
             "ban getValue outside tests",
-            FormatterFunc.NeedsFile { text, file ->
-                if (file.path.contains("commonTest") || file.name == "ObjectCollectionBuilder.kt")
-                    return@NeedsFile text
-                val lines = text.lines()
-                for (line in lines.withIndex()) {
-                    val column = line.value.indexOf("getValue(")
-                    if (column != -1) {
-                        throw IllegalStateException(
-                            "${file.path}:${line.index + 1}:${column + 1} calls getValue"
-                        )
+            object : Serializable, FormatterFunc.NeedsFile {
```

Comment:
This is a workaround for a known issue introduced in the latest version of spotless https://github.com/diffplug/spotless/issues/2387#issuecomment-2576459901